src/main/scala/uk/gov/nationalarchives/tdr/api/http/GraphQLServer.scala

```diff
@@ -77,17 +77,21 @@ object GraphQLServer {
     val ffidMetadataRepository = new FFIDMetadataRepository(db)
```

Comment:
I don't think the following val is needed anymore: `val ffidMetadataMatchesRepository = new FFIDMetadataMatchesRepository(db)`

Reply:
It's not. I think it should be used in `val ffidMetadataService = new FFIDMetadataService(ffidMetadataRepository, new FFIDMetadataMatchesRepository(db), timeSource, uuidSource)` rather than calling new directly in there. I've changed it.